.github/workflows/hermetic_library_generation.yaml

```diff
@@ -19,11 +19,25 @@ on:
 
 env:
   HEAD_REF: ${{ github.head_ref }}
+  REPO_FULL_NAME: ${{ github.event.pull_request.head.repo.full_name }}
 
 jobs:
+  # This step is meant to inline the jobs.if of the next step "library_generation" as
+  # an env var to avoid injection vulnerabilities.
+  # Unfortunately, we can't directly reference `env` in jobs.if
+  # (see https://github.com/actions/runner/issues/2372)
+  # The chosen workaround is to declare an intermediate step that will output the env var
+  prepare-repo-full-name:
+    name: Prepare environment
+    runs-on: ubuntu-latest
+    outputs:
+      REPO_FULL_NAME: ${{ env.REPO_FULL_NAME }}
+    steps:
+      - run: echo "" >> /dev/null # no op - we just need to declare the env var as output
```

Comment:
Is this empty step required?

Reply:
Unfortunately, yes
![image](https://github.com/user-attachments/assets/66746226-8c78-46b8-91a6-6467928ed7e6)

However, we can make it even simpler (`exit 0`)